### ISSUE TITLE
HIghlight key and value for "1Password: Save to 1Password"

### ISSUE THREAD

**Issue by cycle4passion** [1Password/op-vscode#140]
### Summary

let say I have an .env variable like

```js
PUBLIC_APP_URL=127.0.0.1:5000
```
I highlight the `127.0.0.1:5000` part and invoke `1Password: Save in 1Password`. `What do you want to call it?` defaults to save in `url`, and I think thats sounds good. This process creates a standard 1Password item and does not replace the hardcoded selection. I see that not available through `1Password: Get from 1Password`. Thereafter, I figure when asked for the `What do you want to call it?` I put in PUBLIC_APP_URL. 

### Proposed solution

When `1Password: Save to 1Password` is invoked, regex the selection to include the key if highlighted and load that for the default in `What do you want to call it?` Also if the key is not highlighted change the default from `url` to something like `UNDEFINED_KEY` in order to prevent user confusion as I had as above. This change (if key highlighted) would also require the selection be changed before replacing the hardcoded secret with the 1Password replacement.

### Is there a workaround to accomplish this today?
Already know `url` won't work (not in docs?) and manually type a new name.

**Comment by jodyheavener** [comment]
Hi @cycle4passion, thanks for filing this ticket. Apologies it's taken a while to review it. I am trying to reproduce this issue and am having a different result:

- With the line `PUBLIC_APP_URL=127.0.0.1:5000` and the selection `127.0.0.1:5000`
- Running the `1Password: Save in 1Password` command results in no suggestion for "What do you want to call this item?" and "value" for "What do you want this field to be called?"
- Upon completing these fields the selection is replaced with a secret reference

If you're able to, can you try to reproduce your issue and outline the steps you took? Thank you!

**Comment by cycle4passion** [comment]
I agree, its been awhile. As I read what I wrote, I can't even understand it. The steps you propose do work, but I would suggest that they can be error prone. Let me try to lay out my issues.

- If you highlight `https://127.0.0.1:5000` instead of `127.0.0.1:5000` and invoke save to 1Password, it fails but yet creates a 1Password entry. This is probably not what should happen.

- Also, the appropriate name for the 1Password entry is right on the same line, why ask for it again?

What I was trying to indicate in the prior post was, allow highlighting the while line, and regex the name prior to the = and the value following the = .

There may be use cases I am not considering.
